src/main/java/org/takes/misc/Opt.java

```diff
@@ -1,104 +0,0 @@
-/**
```

Comment:
@Happy-Neko why did you remove this class? It was much cleaner than the `Optional` you added